<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialterm</class>
 <widget class="QMainWindow" name="serialterm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serialterm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="txtTerminal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ttyUSB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyUSB1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyUSB2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyACM0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyACM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyACM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyATK0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyATK1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ttyATK2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Terminal 38400-8-n-1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
